### PR TITLE
Explicitly ask a person to describe proposed feature

### PR DESCRIPTION
**Pull request by Xrayez**
There's no part of the template which asks you to describe the proposed feature/enhancement, so people are torn between the 2nd and 3rd questions, often simply paraphrasing the feature instead of focusing on actually describing a feature, not a problem.

See https://github.com/godotengine/godot-proposals/issues/39#issuecomment-528819736.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature---enhacement-proposal.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Proposals not following the template below will be closed immediately.
 
 **Describe the problem or limitation you are having in your project:**
 
-**Describe how this feature / enhancement will help you overcome this problem or limitation:**
+**Describe the feature / enhancement and how will it help to overcome the problem or limitation:**
```
Comment (Xrayez) [inline .github/ISSUE_TEMPLATE/feature---enhacement-proposal.md]:
```suggestion
**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
```
Depending on your mindset. 🙂

Comment (clayjohn) [reply .github/ISSUE_TEMPLATE/feature---enhacement-proposal.md]: The suggested change is much better. The original isn't proper English.

**Comment by clayjohn** [comment]
I think this is a good change! It doesn't take anything away from the template but adds clarity which is always helpful. :)

File: .github/ISSUE_TEMPLATE/feature---enhacement-proposal.md (hunk 1 above)
Comment (Xrayez) [reply .github/ISSUE_TEMPLATE/feature---enhacement-proposal.md]: Learning something new each time, updated.

**Pull request merged**